ios/MissionHub/AppDelegate.m

```diff
@@ -71,7 +75,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   #if FB_SONARKIT_ENABLED
     InitializeFlipper(application);
   #endif
-
+  GAI *gai = [GAI sharedInstance];
+  gai.dispatchInterval = 1;
```

Comment:
@levieggertcru Not sure if I added this in the right spot or not.

Reply:
Hey Christian that should be fine to set it there when application has finished launching.

Reply:
Great, thanks!